blobs.go

```diff
@@ -202,6 +202,7 @@ type CreateOptions struct {
 		// Blob access check will be skipped if set.
 		Stat *Descriptor
 	}
+	TargetMediaType string
```

Comment:
This value is only informational and will be used to allow policy enforcement rather than correctness (digest) or user optimization(mount).